app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/store-selector.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'uiComponent',
+    'Magento_Checkout/js/model/quote',
+    'Magento_Checkout/js/model/step-navigator',
+    'Magento_Checkout/js/model/checkout-data-resolver',
+    'Magento_Checkout/js/action/set-shipping-information',
+], function(
+    $,
+    Component,
+    quote,
+    stepNavigator,
+    checkoutDataResolver,
+    setShippingInformationAction
+) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            template: 'Magento_InventoryInStorePickupFrontend/store-selector',
+            loginFormSelector:
+                '#store-selector form[data-role=email-with-possible-login]',
+        },
+        quoteIsVirtual: quote.isVirtual(),
```

Comment:
What the reason to set value from observable variable to component property? In the current case in the `this.quoteIsVirtual`, you always will have value of `quote.isVirtual()` that was return on the moment of assignments

Reply:
Fixed but basically this was just copied from `shipping.js` file.